backend/tests/apps/api/rest/v0/member_test.py

```
@@ -86,6 +86,34 @@ def test_list_members_with_ordering(self, mock_user_model):
+        mock_user_model.objects.order_by.return_value = mock_ordered_queryset
+        mock_filters.filter.return_value = mock_filtered_queryset
+
+        result = list_members(mock_request, mock_filters, ordering=ordering)
+
+        mock_user_model.objects.order_by.assert_called_with(ordering)
+        mock_filters.filter.assert_called_once_with(mock_ordered_queryset)
+        assert result == mock_filtered_queryset
+
+    def test_list_members_invalid_ordering_returns_422(self):
+        """Test that an invalid ordering field returns a 422 validation error."""
+        from ninja.testing import TestClient
```

Ruff (0.15.2)

[warning] `import` should be at the top-level of a file

(PLC0415)